### PR TITLE
[WIP] Travis CI covers Emacs 27 with Nix

### PR DESCRIPTION
**Pull request by peterbecich**
Uses https://github.com/purcell/nix-emacs-ci

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,87 @@
...
+
+cache:
+  directories:
+  - $HOME/nix.store
+
 sudo: false
+
+jobs:
+  include:
+    - os: linux
```
Comment (conao3) [inline .travis.yml:12]: Can we write `os`, `dist` as a DRY?

**Comment by conao3** [comment]
How is the status?

**Comment by peterbecich** [comment]
Apologies for my lack of response.  I could not get this working.

**Comment by conao3** [comment]
Nothing bad, I wait for next your contribution!

**Pull request closed without merging**